src/derivation.js

```diff
@@ -418,21 +418,43 @@ export const getDerivationModesForCurrency = (
     all.push("legacy_on_segwit");
     all.push("legacy_on_native_segwit");
   }
-  if (!disableBIP44[currency.id]) {
-    all.push("");
-  }
   if (currency.supportsNativeSegwit) {
     all.push("native_segwit");
   }
   if (currency.supportsSegwit) {
     all.push("segwit");
   }
+  if (!disableBIP44[currency.id]) {
+    all.push("");
+  }
```

Comment:
NB: due to this change of order that we agreed on, all the CI tests are going to break and will need to be fixed as well.